README.md

```diff
@@ -89,19 +94,96 @@ int main()
 
   try
   {
-    auto client = BlobClient::CreateFromConnectionString(
-        GetConnectionString(), containerName, blobName);
+    // There are many types of client types, use the one that's appropriate
+    // for your scenario, such as BlobContainerClient.
+    BlobClient client
+        = BlobClient::CreateFromConnectionString(GetConnectionString(), containerName, blobName);
+    BlockBlobClient block = client.AsBlockBlobClient();

```

Comment:
If you get rid of connection strings (which you should), then you should just new up a BlockBlobClient and not have to call AsBlockBlobClient.

Reply:
I also suggest renaming the variable to blockBlobClient; not just block.

Reply:
You could use SharedKeyCredential instead of connection string